pyproject.toml

```diff
@@ -1,25 +1,272 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+
 [project]
-name = "open_pyfus"
-version = "0.1.0"
+name = "openlifu"
+authors = [
+  { name = "Peter Hollender", email = "peterhollender@gmail.com" },
+]
 description = "Openwater Focused Ultrasound Toolkit"
+readme = "README.rst"
+license.file = "LICENSE"
+requires-python = ">=3.11" # TODO: Decrease this version number when resolving #6
+classifiers = [
+  "Development Status :: 1 - Planning",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+#  "Programming Language :: Python :: 3.8", # TODO: Uncomment some of these when resolving #6
+#  "Programming Language :: Python :: 3.9",
+#  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Scientific/Engineering",
+  "Typing :: Typed",
+]
+dynamic = ["version"]
 dependencies = [
-    "xarray",
-    "numpy",
-    "matplotlib",
-    "pandas",
-    "scipy",
-    "vtk",
-    "h5py",
-    "opencv-contrib-python",
-    "crc",
-    "nibabel",
-    "sphinx",
-    "ipykernel"
-]
-
-[tool.setuptools]
-include-package-data = false
-
-[tool.setuptools.packages.find]
-include = ["pyfus*"]
-exclude = ["test*", "env*"]
\ No newline at end of file
+  "xarray",
+  "numpy",
+  "matplotlib",
+  "pandas",
+  "scipy",
+  "vtk",
+  "h5py",
+  "opencv-contrib-python",
+  "crc",
+  "nibabel",
+  "sphinx",
+  "ipykernel",
+  "k-wave-python @ git+https://github.com/peterhollender/k-wave-python@fix_cached_windows_binaries"
```

Comment:
This will prevent publishing to PyPI but we can try to resolve it later. See #7

Reply:
https://github.com/waltsims/k-wave-python/pull/420

Reply:
Now I see that this should point to your master branch and not your `fix_cached_windows_binaries` branch -- I hadn't realized that [your PR](https://github.com/waltsims/k-wave-python/pull/420) is to merge your own master to their master. I'll fix that now

Yes we'll have to deal with this pretty soon since it's a blocker to making a release of openlifu. I opened #7 to track it